homeassistant/components/alarm_control_panel/egardia.py

```diff
@@ -0,0 +1,112 @@
+"""
+Interfaces with Egardia / Woonveilig alarm control panel.
+
+For more details about this platform, please refer to the $
```

Comment:
not sure what you mean. I thought we were supposed to have comments at the top of the file...